cc_licenses/templates/includes/legalcode_30_unported_license.html

```diff
@@ -0,0 +1,308 @@
+{% load license_tags %}
```

Comment:
I think this template looks good. We just need to update the `side-menu` to have anchor tags to the legal code section headers.

Reply:
Thanks, I keep forgetting that!